demo/build.js

```diff
@@ -0,0 +1,52 @@
+import fs from 'fs';
+import esbuild from 'esbuild';
+import chokidar from 'chokidar';
+import glob from 'glob';
+
+const isWatching = (process.argv[2] || '').trim() === '--watch';
+
+// Cleanup existing build
+fs.rmSync('./dist', { recursive: true, force: true });
+fs.mkdirSync('./dist')
+
+// Build CSS
+esbuild.build({
+  entryPoints: ['./src/index.css'],
+  bundle: true,
+  outfile: './dist/index.css',
+  sourcemap: 'inline',
+  watch: isWatching,
+  loader: {
```

Comment:
Is this needed for CSS?

Reply:
Ahh nice catch. Fixed in e7f2a2c9783efbe2e72f6ec5b4975a1a0ac24c5b.